lib/git-hooks.js

```diff
@@ -3,6 +3,12 @@ var util = require('util');
 var spawn = require('child_process').spawn;
 var fs = require('fs');
 var fsHelpers = require('./fs-helpers');
+var execSync = require('child_process').execSync;
+if (!execSync) {
+    execSync = require('runsync').exec;
```

Comment:
maybe I'm dumb, but I haven't catched it yet.)
So the problem you mentioned is with postinstall npm-script? 
And here we just reassign variable so polyfilling it.

You mean that this
```
"preinstall": "if [ $(node -e 'console.log(/0.10/.test(process.version))') = 'true' ]; then npm install runsync; else exit 0; fi"
```
script is buggy?

So we can just always install runsync module